openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -86,6 +86,7 @@ class Meta(object):
     announcement = DateTimeField(null=True)
 
     # URLs
+    banner_image_url = TextField(null=True)
```

Comment:
The only reason I used `null=True` to keep the "convention" seen for the other fields, but I'll adjust the code to use a `default=""` instead as the django convention recommends as well.

Reply:
I see a mix in here where some fields have it and some don't. Since the django convention is well-known, I'm wondering if there's a reason behind it. Unfortunately I don't see anything explained in the comments. Perhaps we should add a comment saying "Not allowing null per django convention; not sure why many TextFields in this model do allow null"

Reply:
Also it's a good idea to review the migration checklist at the end of https://openedx.atlassian.net/wiki/spaces/AC/pages/23003228/Everything+About+Database+Migrations

Reply:
I checked the page, though I'm lack of access to some resources linked in the page. Thank you for the  reading 😇 
It's a good idea about the comment. I'll add that too.

Reply:
Also, I adjusted the migration file (and its name) do you have any other suggestions regarding migrations?

Reply:
Migration looks good to me, thanks.